params/cluster.go

```diff
@@ -20,23 +20,70 @@ type Cluster struct {
 	RendezvousNodes []string `json:"rendezvousnodes"`
 }
 
+type FleetName = string
+type NodeType = string
+
+const (
+	WakuNodes            NodeType = "WakuNodes"
+	DiscV5BootstrapNodes NodeType = "DiscV5BootstrapNodes"
+)
+
 // DefaultWakuNodes is a list of "supported" fleets. This list is populated to clients UI settings.
-var supportedFleets = map[string][]string{
-	FleetShardsStaging: {"enrtree://AI4W5N5IFEUIHF5LESUAOSMV6TKWF2MB6GU2YK7PU4TYUGUNOCEPW@boot.staging.shards.nodes.status.im"},
-	FleetShardsTest:    {"enrtree://AMOJVZX4V6EXP7NTJPMAYJYST2QP6AJXYW76IU6VGJS7UVSNDYZG4@boot.test.shards.nodes.status.im"},
-	FleetWakuSandbox:   {"enrtree://AIRVQ5DDA4FFWLRBCHJWUWOO6X6S4ZTZ5B667LQ6AJU6PEYDLRD5O@sandbox.waku.nodes.status.im"},
-	FleetWakuTest:      {"enrtree://AOGYWMBYOUIMOENHXCHILPKY3ZRFEULMFI4DOM442QSZ73TT2A7VI@test.waku.nodes.status.im"},
+var supportedFleets = map[FleetName]map[NodeType][]string{
+	FleetShardsStaging: {
+		WakuNodes: {
+			"enrtree://AI4W5N5IFEUIHF5LESUAOSMV6TKWF2MB6GU2YK7PU4TYUGUNOCEPW@boot.staging.shards.nodes.status.im",
+		},
+		DiscV5BootstrapNodes: {
```

Comment:
Why not use the config from `config/cli/fleets-<name>.json` rather than again hardcoding in the code?

Reply:
it's a good question. These .json files seem to be used only when running status-go as a service. I imagine we could refactor status-go so the default values come from the config files instead of having them hardcoded. 
wdyt, @cammellos ?

Reply:
I don't mind myself, I don't see the value of having them in JSON files to be honest if they are only used in a single place, this looks more explicit to me and doesn't require a `go generate` (not sure if that's the case with the JSON files), but no strong opinion on my side

Reply:
would you say it's better to get rid of the json files instead?